src/wireviz/DataClasses.py

```diff
@@ -232,6 +232,8 @@ def get_qty_multiplier(self, qty_multiplier: Optional[ConnectorMultiplier]) -> i
             return self.pincount
         elif qty_multiplier == "populated":
             return sum(self.visible_pins.values())
+        elif qty_multiplier == 'unpopulated':
+            return (self.pincount - sum(self.visible_pins.values()))
```

Comment:
```suggestion
            return max(0, self.pincount - sum(self.visible_pins.values()))
```
As already reasoned in https://github.com/wireviz/WireViz/pull/298#issuecomment-1719853330, I suggest limiting this multiplier value to minimum zero.